### ISSUE TITLE
Get Leo's installation path automatically (if installed with pip)

### ISSUE THREAD

**Issue by boltex** [boltex/leointeg#216]
**Ville M. Vainio** proposed that the Leo path setting could be set automatically if Leo was installed with pip.

So a good improvement would be that **on installation, and on upgrade, check with**: ` python -c "import leo; import os; print(os.path.dirname(leo.__file__))"`

(or somesuch) to get the Leo Editor's installation path and set it automatically (if not already set), allowing the user to skip this step before its first LeoInteg usage.

**Comment by maphew** [comment]
This sound small and discrete enough for me to get my feet wet with in getting familiar with Leo Integ. Any pointers where I should start?

**Comment by boltex** [comment]
@maphew For starters, finding a command that could be run in a terminal that would yield the path where Leo is installed would be a good start. Having this I could have leointeg check at startup if the path is set. and if not, it could run this command to see if there's a pip installation of Leo somewhere on the machine. 
(I could run this command in somewhat like how i start a server in leointeg: simply by spawning a new process.)

**Comment by maphew** [comment]
### Scenario 1: Leo is in PATH, for example it's been installed with pipx. Use Leo to report it's own location.

```
echo g.es_print(g.app.loadManager.computeLeoDir()) > tmp_locate_leo_dir.leox
leo-messages --silent --script=tmp_locate_leo_dir.leox
del tmp_locate_leo_dir.leox
```
Output:
```
C:/apps/pipx/venvs/leo/Lib/site-packages/leo
```
As far as I can tell Leo doesn't have a "interpret the following as a script" like `python -c "..."` to enable dispensing with the file writing-reading-delete loop.


### Scenario 2: Python is in PATH and Leo has been installed to it

    python -c "import leo; import os; print(os.path.dirname(leo.__file__))"

## Scenario 3: Multiple pythons managed by Anaconda/Mamba/...

_still working on. I will edit this post when I've got something working_

**Comment by boltex** [comment]
@maphew Wow, you're a command line wizard! Scenario 1 and 2 worked on my machine - so far so good! 

(I'm mostly a javascript dev and so incidentally I know mostly nothing about python's Anaconda/Mamba/etc, environments so many thanks to you for your efforts in trying to automate the 'capture' of Leo's installation folder)

**Comment by boltex** [comment]
@maphew @vivainio This feature is now implemented in the dev branch and will be released in the next update along with Leo 6.7.5. shortly 😄

**Comment by boltex** [comment]
Deployed in 1.0.16
